src/config.json

```diff
@@ -738,7 +738,7 @@
     "axle.design",
     "cactus.chat",
     "nfinity.space",
-    "catctus.io"
```

Comment:
Oh, just noticed that you were deleting this entry. I assume this was a mistake.